plugins/org.jboss.tools.openshift.client/.project

```diff
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
dito, why remove this? you'll force developers to import as maven project and have maven triggered upon build which brings no real benefit here: this is an Eclipse plugin, what else than Eclipse would you use to develop it?